examples/sandbox_resources.py

```diff
@@ -0,0 +1,32 @@
+import requests
+from nkms.characters import Ursula
+from nkms.network.node import NetworkyStuff
+from nkms.crypto.powers import SigningPower, EncryptingPower
+
+
+class SandboxNetworkyStuff(NetworkyStuff):
+
+    def __init__(self, ursulas):
+        self.ursulas = ursulas
+
+    def find_ursula(self, contract=None):
```

Comment:
Would we have this fort of simplified API not only in demos?

Reply:
The stimulus for this sprint was to work out the simplest, highest-level set of commands that an external developer could write into their codebase, and then walk through these in alicebob-grant-and-line-by-line-PRE.py. This file can then serve as a 'beach head' for customers, who can trial relatively basic functions like ALICE.grant, BOB.join_policy and BOB.retrieve before moving on to greater customization, attempts to economize their usage, more complex sharing workflows, etc. 

So, I think the answer is yes – this simplified API is helpful beyond a demo.

Reply:
:+1: